command/login.go

```diff
@@ -43,15 +43,12 @@ var (
 )
 
 func runLogin(cmd *Command, args []string) {
-	var endpoint ForceEndpoint = EndpointProduction
+	var endpoint string = "https://login.salesforce.com"
```

Comment:
Why not `endpoint := "https://login.salesforce.com"`?

Reply:
Inertia :smiley:

Updated in 114a64b297ee1e3a8238bea2ceb6f949ea05c092.